config/initializers/hyrax.rb

```diff
@@ -0,0 +1,4 @@
+# Use this to override any Hyrax configuration from the Knapsack
```

Comment:
The generator itself made the index.rb file so I don't think we want this included in the pull request.

Reply:
that was not the case for me, i had to make sure the file was already exising